packages/app/src/cli/services/versions-list.test.ts

```diff
@@ -45,88 +61,95 @@ describe('versions-list', () => {
     vi.mocked(partnersRequest).mockResolvedValueOnce(emptyResult)
 
     // When
-    await versionList({app, apiKey: 'apiKey'})
+    await versionList({
+      app,
+      reset: false,
+      commandConfig: {runHook: vi.fn(() => Promise.resolve({successes: []}))} as unknown as Config,
+    })
 
     // Then
     expect(outputMock.info()).toMatchInlineSnapshot(`"No app versions found for this app"`)
   })
 
-  test('throw error when there is no app', async () => {
-    // Given
-    const app = await testApp({})
-    vi.mocked(partnersRequest).mockResolvedValueOnce({app: null})
-
-    // When
-    const output = versionList({app, apiKey: '1235'})
-
-    // Then
-    await expect(output).rejects.toThrow('Invalid API Key: 1235')
-  })
-
-  test('use apikey from env if its not passed via flag', async () => {
+  test('show currently used config info', async () => {
     // Given
     const app = await testApp({})
-    vi.mocked(ensureAuthenticatedPartners).mockResolvedValue('token')
-    vi.mocked(getAppIdentifiers).mockReturnValue({app: 'env-app-id'})
     vi.mocked(partnersRequest).mockResolvedValueOnce(emptyResult)
 
     // When
-    await versionList({app})
+    await versionList({
+      app,
+      reset: false,
+      commandConfig: {runHook: vi.fn(() => Promise.resolve({successes: []}))} as unknown as Config,
+    })
 
     // Then
-    expect(vi.mocked(partnersRequest)).toHaveBeenCalledWith(expect.anything(), 'token', {apiKey: 'env-app-id'})
-    expect(vi.mocked(fetchOrCreateOrganizationApp)).not.toHaveBeenCalled()
+    expect(renderCurrentlyUsedConfigInfo).toHaveBeenCalledWith({
+      org: 'name of org 1',
+      appName: 'app-title',
+      configFile: undefined,
+    })
   })
 
-  test('use client_id from linked toml if no env ids or flags', async () => {
```

Comment:
It's a bit difficult to parse the diff, but it looks like this test and the next were deleted. Was that on purpose?

Reply:
Oh yeah, the diff is even confusing me 😅 Yeah, I deleted these tests because this PR removes [the `getAppApiKey()` function](https://github.com/Shopify/cli/blob/04e6341280afad9e8dc1d735d8c1050d0c9a1932/packages/app/src/cli/services/versions-list.ts#L83-L90) (which is what I believe is being tested in these tests).